fastify.d.ts

```diff
@@ -261,11 +266,13 @@ declare namespace fastify {
      * Apply the given middleware to all incoming requests
      */
     use(middleware: Middleware<HttpServer, HttpRequest, HttpResponse>): void
+    use(middleware: express.RequestHandler): void
```

Comment:
We do not expose the enhanced version of the request/response object of express.
Is this correct?

Reply:
I agree about this... The `cors` library currently types itself as express middleware, so I added that really just to keep the tests passing.
I'll work on removing the dep on express...